feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Store, StoreModule } from '@ngrx/store';
+import { StateWithAsm } from '@spartacus/asm/core';
+import { AsmFacade, ASM_FEATURE } from '@spartacus/asm/root';
+import { ActiveCartFacade, MultiCartFacade } from '@spartacus/cart/base/root';
+import { BaseSiteService, User } from '@spartacus/core';
+import { UserAccountFacade } from '@spartacus/user/account/root';
+import { Observable, of } from 'rxjs';
+import * as fromReducers from '../../core/store/reducers/index';
+import { AsmBindCartComponent } from './asm-bind-cart.component';
+
+class MockActiveCartService {
+  getActiveCartId(): Observable<string> {
+    return of();
+  }
+}
+
+const baseSite = 'test-site';
+class MockBaseSiteService {
+  getActive(): Observable<string> {
+    return of(baseSite);
+  }
+}
+
+@Pipe({
+  name: 'cxTranslate',
+})
+class MockTranslatePipe implements PipeTransform {
+  transform(): any {}
+}
+
+class MockUserAccountFacade implements Partial<UserAccountFacade> {
+  get(): Observable<User> {
+    return of({});
+  }
+}
+
+class MockMultiCartFacade {
+  loadCart(_cartId: string, _userId: string): void {}
+}
+
+class MockAsmService {
+  bindCart(_cartId: string, _customerId: string): Observable<unknown> {
+    return of(null);
+  }
+}
+
+describe('AsmBindCartComponent', () => {
+  let component: AsmBindCartComponent;
+  let fixture: ComponentFixture<AsmBindCartComponent>;
+  let asmFacade: AsmFacade;
+  let multiCartFacade: MultiCartFacade;
+  let activeCartFacade: ActiveCartFacade;
+  let store: Store<StateWithAsm>;
+  let userService: UserAccountFacade;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [
+        StoreModule.forRoot({}),
+        StoreModule.forFeature(ASM_FEATURE, fromReducers.getReducers()),
+      ],
+      declarations: [AsmBindCartComponent, MockTranslatePipe],
```

Comment:
If all the services/dependencies are mocked, you should not need to inject the store in this test.

Reply:
Unit test fails if removed